packages/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -169,7 +126,63 @@ const useRootStyles = makeStyles({
       color: theme.alias.color.neutral.neutralForeground2BrandPressed,
     },
   }),
-  disabled: theme => ({
+
+  // Shape variations
+  circular: theme => ({
+    borderRadius: theme.global.borderRadius.circular,
+  }),
+  rounded: {
+    /* The borderRadius rounded styles are handled in the size variations */
+  },
+  square: theme => ({
+    borderRadius: theme.global.borderRadius.none,
+  }),
+
+  // Size variations
+  small: theme => ({
+    gap: buttonSpacing.smaller,
+    padding: `0 ${buttonSpacing.medium}`,
+
+    height: '24px',
```

Comment:
Not for this PR, but we should probably adjust this to use padding/lineheight so the button text can wrap OK (I'll add it to the a11y issue 😄)